packages/graphql/src/api-v6/schema-generation/schema-types/StaticSchemaTypes.ts

```diff
@@ -180,6 +180,39 @@ class StaticFilterTypes {
         });
     }
 
+    public getBooleanListWhere(nullable: boolean): InputTypeComposer {
+        if (nullable) {
+            return this.schemaBuilder.getOrCreateInputType("StringListWhereNullable", () => {
+                return {
+                    fields: {
+                        equals: "[Boolean]",
```

Comment:
```suggestion
                        equals: list(GraphQLBoolean.name),
```
I would import GraphQLBoolean from the GraphQL spec as for the others below to avoid typos and making consistent with other custom built-in Scalars!